test/helpers/controller.go

```diff
@@ -13,6 +14,17 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
+// RequestBody return a request body
+func RequestBody(data map[string]string) (body io.Reader) {
```

Comment:
Minor naming improvement since this method returns a value and it's not clear what this method is doing just based on the current name:

```suggestion
func GenerateRequestBody(data map[string]string) (body io.Reader) {
```

It would be consistent with `GetResponseBody`.

Reply:
@olivierobert ✅  renamed on [ef1b55c](https://github.com/carryall/go-google-scraper-challenge/pull/26/commits/ef1b55c69532b4ced9737d33f6c8d4d497ed498a)